cloudimg/common.py

```diff
@@ -33,7 +34,7 @@ class PublishingMetadata(object):
 
     def __init__(self, image_path, image_name, description=None,
                  container=None, arch=None, virt_type=None,
-                 root_device_name=None, volume_type=None,
+                 root_device_name=None, volume_type=None, uefi_support=None,
```

Comment:
Sorry for missing this earlier.  This change is potentially backwards-incompatible as you're adding a new parameter into the middle of the signature.

If anyone is constructing a PublishingMetadata using positional arguments, this commit will change the meaning of the 9th argument from `accounts` to `uefi_support` which would break that code.

You might hope that all callers are using keyword arguments and not positional arguments, but it can't be safely assumed since the code isn't designed to enforce that.

Please only add arguments at the end instead so we can be sure that a new release isn't going to silently change the meaning of existing code.